homeassistant/components/weather/darksky.py

```diff
@@ -19,7 +19,7 @@
 import homeassistant.helpers.config_validation as cv
 from homeassistant.util import Throttle
 
-REQUIREMENTS = ['python-forecastio==1.3.5']
+REQUIREMENTS = ['python-forecastio==1.4.0']
```

Comment:
1. You need to update the requirement in `components/sensor/darksky` as well. Otherwise we'll have a version conflict.
2. Please also run the `script/gen_requirements_all.py` script to update `requirements_all.txt` and `requirements_test_all.txt`
3. This version bump doesn't change anything for us right now: https://github.com/ZeevG/python-forecast.io/commits/master; So I'd advise to keep the old version until @nodomain can get his language configuration option merged. See https://github.com/ZeevG/python-forecast.io/pull/53

Reply:
It is merged since today 🕺

Reply:
@nodomain I meant a PR in the *Home Assistant repo* that adds lang capabilities. Without the PR for lang in Home Assistant, this version bump isn't really necessary. Never change a running system unless there's a reason to :)

Reply:
Ah. Edge cases of language with all their misunderstandings ;-)